nf_core/pipeline-template/nextflow.config

```diff
@@ -274,7 +274,18 @@ dag {
 
 manifest {
     name            = '{{ name }}'
-    author          = """{{ author }}"""
+    author          = """{{ author }}""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: '{{ author }}',
```

Comment:
what happens in case of multiple authors?

Reply:
you can add another map to the list of maps

Reply:
ah sorry do you mean the `name` only? should we also make it empty and let people complete?

Reply:
Then I would leave it empty, because we are allowing both notations for now

Reply:
leaving it empty feels weird... why not have a long string in author and then an array for contributor where we split the string?

 we probably should also update the llnting to be able to handle empty author fields.

Reply:
I pushed a change using a loop, what do you think?

Reply:
yes, I think that should be good.